src/classes/BotManager.ts

```diff
@@ -35,7 +35,7 @@ export = class BotManager {
 
         this.socket.on('connect', () => {
             log.debug('Connected to socket server');
-            this.socket.emit('authentication', process.env.PRICESTF_API_TOKEN);
+            this.socket.emit('authentication', process.env.PRICESTF_API_TOKEN || undefined);
```

Comment:
I guess I thought that it should be undefined anyway, but people probably just copy-paste the config and don't fill in the api key.

Reply:
I mean most just change the filename, lol

Reply:
yeah.. but leaving it empty will cause an infinite loop of connecting and then disconnected to socket server. I'm not sure why.

Reply:
it is because the socket keeps getting disconnected, it will then reconnect because of the code below